js/src/pages/price-benchmark/price-benchmark-suggestions.js

```diff
@@ -122,6 +124,14 @@ const PriceBenchmarkSuggestions = () => {
 	const { suggestions, hasFinishedResolution } =
 		usePriceBenchmarkSuggestions();
 
+	if ( suggestions.length === 0 ) {
+		return (
+			<EmptyTable>
```

Comment:
@ankitrox Maybe we can `EmptyTable` within `EmptyMetricsNotice` since it's being used within tables only.